src/java.base/share/classes/javax/crypto/SealedObject.java

```diff
@@ -421,8 +421,11 @@ private Object unseal(Key key, String provider)
 
     /**
      * Restores the state of the SealedObject from a stream.
+     *
      * @param s the object input stream.
-     * @exception NullPointerException if s is null.
+     * @throws IOException if an I/O error occurs
+     * @throws ClassNotFoundException if a serialized class cannot be loaded
+     * @throws NullPointerException if s is null.
```

Comment:
Remove period for consistency with other throws. Actually, you probably don't need to say that it throws NPE. I don't see that any other readObject method declares that, even if they do throw NPE if the stream is null. Seems like something that should just be assumed or does not happen under normal circumstances.

Reply:
It did seem really strange to me.  I'll remove.